thunder/core/prims.py

```diff
@@ -3763,6 +3764,65 @@ def sort_meta(a: TensorProxy, /, dim: int, descending: Number, sorted: Number) -
 sort = make_prim(PrimIDs.SORT, "sort", meta=sort_meta)
 
 
+def _grouped_mm_meta(
+    a: TensorProxy, b: TensorProxy, offs: None | TensorProxy = None, bias: None | TensorProxy = None, dtype=None
```

Comment:
nit-picking. type annotation for `dtype`.

I don't remember how this file imports `thunder.core.dtypes` though
```suggestion
    a: TensorProxy, b: TensorProxy, offs: None | TensorProxy = None, bias: None | TensorProxy = None, dtype: dtyps.dtype | None = None
```